### ISSUE TITLE
Cannot Install v1.2.2

### ISSUE THREAD

**Issue by briprowe** [copiousfreetime/hitimes#43]
Notice that my home directory is `/Users/browe` and that it's trying to link files in `-L/Users/mpapis/.sm/pkg/active/lib'`.

```
$ gem install hitimes -v '1.2.2'
Building native extensions.  This could take a while...
ERROR:  Error installing hitimes:
	ERROR: Failed to build gem native extension.

    /Users/browe/.rvm/rubies/ruby-2.0.0-p247/bin/ruby extconf.rb
creating Makefile

make "DESTDIR="
compiling hitimes.c
compiling hitimes_instant_clock_gettime.c
compiling hitimes_instant_osx.c
compiling hitimes_instant_windows.c
compiling hitimes_interval.c
compiling hitimes_stats.c
linking shared-object hitimes/2.0/hitimes.bundle
ld: warning: directory not found for option '-L/Users/mpapis/.sm/pkg/active/lib'
ld: framework not found CoreServices
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [hitimes.bundle] Error 1


Gem files will remain installed in /Users/browe/.rvm/gems/ruby-2.0.0-p247/gems/hitimes-1.2.2 for inspection.
Results logged to /Users/browe/.rvm/gems/ruby-2.0.0-p247/gems/hitimes-1.2.2/ext/hitimes/c/gem_make.out
```

The offending lines seem to be in the Makefile:

```
$ find /Users/browe/.rvm/gems/ruby-2.0.0-p247/gems/hitimes-1.2.2 | xargs -I{} grep -Hn '/Users/mpapis' {} 2> /dev/null
/Users/browe/.rvm/gems/ruby-2.0.0-p247/gems/hitimes-1.2.2/ext/hitimes/c/Makefile:81:CFLAGS   = $(CCDLFLAGS) -O3 -I/Users/mpapis/.sm/pkg/active/include -fPIC -pipe -DUSE_INSTANT_OSX=1 -Wall $(ARCH_FLAG)
/Users/browe/.rvm/gems/ruby-2.0.0-p247/gems/hitimes-1.2.2/ext/hitimes/c/Makefile:86:ldflags  = -L. -Z -L/Users/mpapis/.sm/pkg/active/lib -L/usr/lib -fPIC -Bstatic -fstack-protector -framework CoreServices
```

I'm running Mac OS X Yosemite.

If you need further information, let me know.

**Comment by briprowe** [comment]
If this was working in the past, my guess is the problem lies in a transitive dependency.

**Comment by briprowe** [comment]
For good measure, here's my bundler version:

```
$ bundle --version
Bundler version 1.3.5
```

And my Ruby version:

```
$ ruby --version
ruby 2.0.0p247 (2013-06-27 revision 41674) [x86_64-darwin12.3.0]
```

**Comment by briprowe** [comment]
Here's an interesting find:

```
$ rvm config-get | grep mp
CFLAGS: -O3 -I/Users/mpapis/.sm/pkg/active/include -fPIC -pipe
CXXFLAGS:  -O3 -fno-fast-math -ggdb3 -Wall -Wextra -Wno-unused-parameter -Wno-parentheses -Wno-long-long -Wno-missing-field-initializers -Wunused-variable -Wpointer-arith -Wwrite-strings -Wdeclaration-after-statement -Wshorten-64-to-32 -Wimplicit-function-declaration
LDFLAGS: -L. -Z -L/Users/mpapis/.sm/pkg/active/lib -L/usr/lib -fPIC -Bstatic -fstack-protector
cflags:  -O3 -fno-fast-math -ggdb3 -Wall -Wextra -Wno-unused-parameter -Wno-parentheses -Wno-long-long -Wno-missing-field-initializers -Wunused-variable -Wpointer-arith -Wwrite-strings -Wdeclaration-after-statement -Wshorten-64-to-32 -Wimplicit-function-declaration
configure_args:  '--prefix=/Users/mpapis/.rvm/rubies/ruby-2.0.0-p247' '--enable-load-relative' '--with-static-linked-ext' '--without-ext=dl/win32,fiddle/win32,tk/tkutil,win32ole,-test-/win32/dln,-test-/win32/fd_setsize' '--sysconfdir=/etc' '--disable-install-doc' '--without-tcl' '--without-tk' '--disable-shared' '--with-arch=x86_64' 'CC=/usr/bin/clang' 'CFLAGS=-O3 -I/Users/mpapis/.sm/pkg/active/include -fPIC' 'LDFLAGS=-Z -L/Users/mpapis/.sm/pkg/active/lib -L/usr/lib -fPIC -Bstatic'
cxxflags:  -O3 -fno-fast-math -ggdb3 -Wall -Wextra -Wno-unused-parameter -Wno-parentheses -Wno-long-long -Wno-missing-field-initializers -Wunused-variable -Wpointer-arith -Wwrite-strings -Wdeclaration-after-statement -Wshorten-64-to-32 -Wimplicit-function-declaration
warnflags: -Wall -Wextra -Wno-unused-parameter -Wno-parentheses -Wno-long-long -Wno-missing-field-initializers -Wunused-variable -Wpointer-arith -Wwrite-strings -Wdeclaration-after-statement -Wshorten-64-to-32 -Wimplicit-function-declaration
[~/blog (master)]
$ rvm config-get | grep mpapis
CFLAGS: -O3 -I/Users/mpapis/.sm/pkg/active/include -fPIC -pipe
LDFLAGS: -L. -Z -L/Users/mpapis/.sm/pkg/active/lib -L/usr/lib -fPIC -Bstatic -fstack-protector
configure_args:  '--prefix=/Users/mpapis/.rvm/rubies/ruby-2.0.0-p247' '--enable-load-relative' '--with-static-linked-ext' '--without-ext=dl/win32,fiddle/win32,tk/tkutil,win32ole,-test-/win32/dln,-test-/win32/fd_setsize' '--sysconfdir=/etc' '--disable-install-doc' '--without-tcl' '--without-tk' '--disable-shared' '--with-arch=x86_64' 'CC=/usr/bin/clang' 'CFLAGS=-O3 -I/Users/mpapis/.sm/pkg/active/include -fPIC' 'LDFLAGS=-Z -L/Users/mpapis/.sm/pkg/active/lib -L/usr/lib -fPIC -Bstatic'
```

I'm using rvm version 1.26.11

I don't know anything about ruby, so I don't know how to fix this. I recognize it's not a bug in hitimes, so feel free to close this.

**Comment by copiousfreetime** [comment]
Yes, this is not a bug in hitimes, this is a result of rvm and/or rvm installing ruby. I would suggest that an upgrade of rvm might fix this, or a reinstall of ruby 2.0 or an install of ruby 2.1. See bugs #27 #28 and #35

**Comment by copiousfreetime** [comment]
@briprowe It looks like you can fix it if you reinstall ruby and have it ignore the binary. See https://github.com/rvm/rvm/issues/3360. See the final comments there form @mpapis
